src/js/index.js

```diff
@@ -11,30 +11,32 @@ var model = require('./model');
 model.init();
 
 // Initialize components
-var Sidebars = require('./components');
+var ui = require('./components');
 
 var g = model.Scene.child('marks.group'),
+    g2 = g.child('marks.group'),
+    g3 = g2.child('marks.group'),
```

Comment:
I think that's fine.  I just wanted to make sure it was intentional and not debugging code.